DBZ-4762 Document excluded built-in schemas (#3435)
debezium/debezium: documentation/modules/ROOT/pages/connectors/oracle.adoc

The {prodname} Oracle connector is compatible with Oracle installed as a standalone instance.
endif::product[]

> === Built-in excluded schemas

Reviewer: ```suggestion
=== Schema types excluded from capture 
```

> The {prodname} Oracle connector automatically excludes capturing changes from the following schemas:

Reviewer: ```suggestion
When the {prodname} Oracle connector captures tables, it automatically excludes tables from the following schemas:
```

> `appqossys`, `audsys`, `ctxsys`, `dvsys`, `dbsfwuser`, `dbsnmp`, `qsmadmin_internal`, `lbacsys`, `mdsys`, `ojvmsys`, `olapsys`, `orddata`, `ordsys`, `outln`, `sys`, `system`, `wmsys`, and `xdb`.

Reviewer: ```suggestion
* `appqossys`
* `audsys`
* `ctxsys`
* `dvsys`
* `dbsfwuser`
* `dbsnmp` 
* `qsmadmin_internal`
* `lbacsys`
* `mdsys`
* `ojvmsys`
* `olapsys`
* `orddata`
* `ordsys`
* `outln`
* `sys`
* `system`
* `wmsys`
* `xdb`
```

> In order to guarantee that a table is captured by the connector, please be sure to any other schema.

Reviewer: To enable the connector to capture changes from a table, the table must use a schema that is not named in the preceding list.
